[2025-03-27T16:46:27.497+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-27T16:46:27.535+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_inaturalist_data_atropelada.process_data scheduled__2025-03-27T16:00:00+00:00 [queued]>
[2025-03-27T16:46:27.581+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_inaturalist_data_atropelada.process_data scheduled__2025-03-27T16:00:00+00:00 [queued]>
[2025-03-27T16:46:27.590+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 4
[2025-03-27T16:46:27.627+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): process_data> on 2025-03-27 16:00:00+00:00
[2025-03-27T16:46:27.674+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'get_inaturalist_data_atropelada', 'process_data', 'scheduled__2025-03-27T16:00:00+00:00', '--job-id', '167', '--raw', '--subdir', 'DAGS_FOLDER/inaturalist_dag.py', '--cfg-path', '/tmp/tmpxk_qsxue']
[2025-03-27T16:46:27.671+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=376) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-27T16:46:27.681+0000] {standard_task_runner.py:105} INFO - Job 167: Subtask process_data
[2025-03-27T16:46:27.688+0000] {standard_task_runner.py:72} INFO - Started process 380 to run task
[2025-03-27T16:46:27.943+0000] {task_command.py:467} INFO - Running <TaskInstance: get_inaturalist_data_atropelada.process_data scheduled__2025-03-27T16:00:00+00:00 [running]> on host d54220454ad9
[2025-03-27T16:46:28.375+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='get_inaturalist_data_atropelada' AIRFLOW_CTX_TASK_ID='process_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-27T16:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-27T16:00:00+00:00'
[2025-03-27T16:46:28.392+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-27T16:46:28.402+0000] {taskinstance.py:301} INFO - No observations to process, skipping tasks
[2025-03-27T16:46:28.528+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-27T16:46:28.531+0000] {taskinstance.py:352} INFO - Marking task as SKIPPED. dag_id=get_inaturalist_data_atropelada, task_id=process_data, run_id=scheduled__2025-03-27T16:00:00+00:00, execution_date=20250327T160000, start_date=20250327T164627, end_date=20250327T164628
[2025-03-27T16:46:28.679+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-27T16:46:28.792+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-27T16:46:28.826+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
